src/Algolia.Search/Models/ApiKeys/RestoreApiKeyResponse.cs

```diff
@@ -0,0 +1,76 @@
+/*
+* Copyright (c) 2018 Algolia
```

Comment:
As much as I hate copyright headers (flashback to early 2000s enterprise development) 
It's 2019 😉